bundles/io/org.openhab.io.caldav/src/main/java/org/openhab/io/caldav/CalDavQuery.java

```diff
@@ -27,6 +27,7 @@
     private Sort sort;
     private String filterName;
     private List<String> filterCategory;
+    private boolean filterCategoryMatchesAny = false;
```

Comment:
Any reason this member should receive an initial value and not the other members?